canvas_cli/main.py

```diff
@@ -51,6 +58,37 @@ def get_or_create_config_file() -> Path:
     return config_path
 
 
+@app.command()
+def emit(
+    event_fixture: str,
+    plugin_runner_port: Annotated[
+        str, typer.Option(help="Port of your locally running plugin runner")
+    ] = "50051",
+) -> None:
+    """
+    Grab an event from a fixture file and send it your locally running plugin-runner process.
+    Any resultant effects will be printed.
+
+    Valid fixture files are newline-delimited JSON, with each containing the keys `EventType`, `target`, and `context`. Some fixture files are included in the canvas-plugins repo.
+    """
+    # Grab a random event from the fixture file ndjson
+    lines = open(event_fixture).read().splitlines()
```

Comment:
I think that syntax leaves the file handle open.

```suggestion
    with open(event_fixture) as file_:
        lines = file_.read().splitlines()
```

Reply:
you can do it without a context if you use pathlib, e.g. `Path(event_fixture).read_all().splitlines()`

Reply:
Looks like it's `Path(event_fixture).read_text().splitlines()`